packages/components/src/internal/components/pagination/Pagination.tsx

```diff
@@ -31,95 +31,104 @@ export interface PaginationProps extends PaginationData {
 }
 
 const PAGINATION_METRIC_AREA = 'pagination';
-
-export class Pagination extends PureComponent<PaginationProps> {
-    static defaultProps = {
-        pageSizes: [20, 40, 100, 250, 400],
-    };
-
-    onLoadFirstPage = () => {
+const DEFAULT_PAGE_SIZES = [20, 40, 100, 250, 400];
```

Comment:
- [ ] Issue 51421: Inconsistent behavior when hitting `Shift` + `End` in the editable grid
    - Not yet verified (don't have `End` key on laptop -- will verify later)
- [x] Issue 51909: Hit Selection Dialog should show field-modal__field_dot next to fields that have criteria
- [ ] Issue 51897: Grid loses paging control and shows "No data" when samples in a filtered view are updated
    - Did not manual test -- will leave to reporter / assigned tester

Reply:
I've since verified Issue 51421.